GNUmakefile

```diff
@@ -22,6 +22,11 @@ clean:
 release:
 	@git tag ${NEXT_VERSION} && git push --mirror
 	@echo "Pushed ${NEXT_VERSION}"
+	GOPROXY=https://proxy.golang.org GO111MODULE=on go get github.com/jfrog/terraform-provider-artifactory@v${NEXT_VERSION}
```

Comment:
`go get` has been deprecated for installing binaries since Go 1.17 and will be become impossible in Go 1.18. If you are using Go 1.16 or above, you should use instead: `go install`

Reply:
Let's cross that bridge when we get there. Important note

Reply:
`go get` is fine to use in this context, even in Go 1.18. This command is to send a download request to https://proxy.golang.org and refreshes its cache.